src/server.jl

```diff
@@ -52,7 +52,7 @@ function run_server(app::DashApp,
     start_server = () -> begin
         handler = make_handler(app);
         server = Sockets.listen(get_inetaddr(host, port))
-        task = @async HTTP.serve(handler, host, port; server = server, verbose = true)
+        task = @async HTTP.serve(handler, host, port; server = server, verbose = debug==true)
```

Comment:
Personally I would set `verbose=false` even in under `run_server(app; debug=true)` 

I'd say logs like

```
┌ Debug: startread called
└ @ HTTP.Servers ~/.julia/packages/HTTP/z8l0i/src/Servers.jl:427
┌ Debug: invoking handler
└ @ HTTP.Servers ~/.julia/packages/HTTP/z8l0i/src/Servers.jl:446
┌ Debug: closeread
└ @ HTTP.Servers ~/.julia/packages/HTTP/z8l0i/src/Servers.jl:452
┌ Debug: closewrite
└ @ HTTP.Servers ~/.julia/packages/HTTP/z8l0i/src/Servers.jl:454
┌ Debug: startread called
└ @ HTTP.Servers ~/.julia/packages/HTTP/z8l0i/src/Servers.jl:427
┌ Debug: invoking handler
└ @ HTTP.Servers ~/.julia/packages/HTTP/z8l0i/src/Servers.jl:446
┌ Debug: closeread
└ @ HTTP.Servers ~/.julia/packages/HTTP/z8l0i/src/Servers.jl:452
┌ Debug: closewrite
└ @ HTTP.Servers ~/.julia/packages/HTTP/z8l0i/src/Servers.jl:454
┌ Debug: startread called
└ @ HTTP.Servers ~/.julia/packages/HTTP/z8l0i/src/Servers.jl:427
┌ Debug: invoking handler
└ @ HTTP.Servers ~/.julia/packages/HTTP/z8l0i/src/Servers.jl:446
┌ Debug: closeread
└ @ HTTP.Servers ~/.julia/packages/HTTP/z8l0i/src/Servers.jl:452
┌ Debug: closewrite
└ @ HTTP.Servers ~/.julia/packages/HTTP/z8l0i/src/Servers.jl:454
┌ Debug: startread called
└ @ HTTP.Servers ~/.julia/packages/HTTP/z8l0i/src/Servers.jl:427
┌ Debug: invoking handler
└ @ HTTP.Servers ~/.julia/packages/HTTP/z8l0i/src/Servers.jl:446
┌ Debug: closeread
└ @ HTTP.Servers ~/.julia/packages/HTTP/z8l0i/src/Servers.jl:452
```

are pretty much useless for Dash.jl users. Thoughts?

Reply:
yes I think it's overly verbose for Dash and useless

Reply:
thanks for your input @mind6 !